lib/predictions/supervisor.ex

```diff
@@ -8,19 +8,21 @@ defmodule Predictions.Supervisor do
 
   use Supervisor
 
+  @predictions_phoenix_pub_sub Application.compile_env!(:dotcom, [:predictions_phoenix_pub_sub])
+
   def start_link(_) do
     Supervisor.start_link(__MODULE__, [])
   end
 
   @impl Supervisor
   def init(_) do
     children = [
-      {Phoenix.PubSub, [name: Predictions.PubSub]},
+      {Phoenix.PubSub, [name: @predictions_phoenix_pub_sub]},
```

Comment:
same suggestion re: dynamic vs static configuration

Reply:
We are moving away from DI and toward using Mox. That's 50% of the purpose of this PR.